server/backend/migration/2023090410000000.go

```
@@ -0,0 +1,58 @@
+package migration
+
+import (
+	"database/sql"
+
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"github.com/guregu/null"
+	"gorm.io/gorm"
+)
+
+// migrate2023090410000000
+// migrates the crontap from kino to movie crontab
+func (m TumDBMigrator) migrate2023090410000000() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "2023090410000000",
+		Migrate: func(tx *gorm.DB) error {
+			// modify the crontab
+			if err := tx.Delete(&model.Crontab{}, "type = ?", "kino").Error; err != nil {
+				return err
+			}
+			if err := SafeEnumRollback(tx, &model.Crontab{}, "type", "kino"); err != nil {
+				return err
+			}
+			if err := SafeEnumMigrate(tx, &model.Crontab{}, "type", "movie"); err != nil {
+				return err
+			}
+			if err := tx.Create(&model.Crontab{
```

you could just return the err here directly